src/framing/mod.rs

```diff
@@ -101,7 +93,7 @@ impl MLSPlaintext {
     pub fn as_application_message(&self) -> Result<&[u8], MLSPlaintextError> {
```

Comment:
(not the correct line)

Shouldn't `verify()` return an error as well instead of a `bool`?

Reply:
It probably should, but again a logic change I wouldn't want to do in this PR (updated #83).